ci-operator/step-registry/sandboxed-containers-operator/get-kata-rpm/sandboxed-containers-operator-get-kata-rpm-ref.yaml

```diff
@@ -11,13 +11,11 @@ ref:
       name: sandboxed-containers-operator-ci-secrets
       namespace: test-credentials
   env:
-  - name: INSTALL_KATA_RPM
-    default: "false"
-  - name: KATA_RPM_VERSION
-    default: "3.13.0-1.rhaos4.18.el9"
-  - name: KATA_RPM_BUILD_MD5SUM
-    default: "da576553a2d5925b67ddaf0591454e48"
-  - name: KATA_RPM_BUILD_TASK
-    default: ""
+    - name: INSTALL_KATA_RPM
+      default: "false"
+    - name: KATA_RPM_BUILD_TASK
+      default: ""
+    - name: KATA_RPM_VERSION
+      default: "3.13.0-3.rhaos4.18.el9"
```

Comment:
This doesn't seem related to this change, was it generated by `make jobs`? I'm wondering if it's some jq versioning issue...

Reply:
I included it to run

Reply:
To run what? Could you please only remove the `MD5SUM` and leave the formatting as is? You can then run the tests manually by workflow name...